thunder/torch/__init__.py

```diff
@@ -5465,6 +5466,8 @@ def register_default_torch_op(torchfn: Callable, torch_module):
 
     op = ex.register_operator(torchfn_name, module=torch_module, meta=fn_meta)
     ex.register_implementation(sym, op, checker=_always_executable)
+    if getattr(sys.modules["thunder.torch"], torchfn_name, None) is None:
+        object.__setattr__(sys.modules["thunder.torch"], torchfn_name, sym)
```

Comment:
Nifty!
- I think `hasattr` would be a better check,
- do you know if there are cases where this does is not the case? Ideally we could make it an assert instead of an `if`, so we immediately know if there is duplication.
- I think we may need to handle the case of submodules (.nn,, .linear, ...) by registering them in their own namespece (and adjusting module.